### ISSUE TITLE
Window icons too large for small terminals

### ISSUE THREAD

**Issue by adius** [marionebl/svg-term-cli#24]
![bug](https://user-images.githubusercontent.com/969983/35194853-1cd86dda-feb2-11e7-9301-fb69765481b5.png)

**Comment by marionebl** [comment]
Please provide the command + cast to reproduce

**Comment by adius** [comment]
https://asciinema.org/a/YsboQmbF6dA5KcJnx7rxJhLlv

```
svg-term \
                          --no-cursor \
                          --at 99999 \
                          --window \
                          --term iterm2 \
                          --profile ../../dotfiles/terminal/adius.itermcolors \
                          < bug.json \
                          > bug.svg
```

**Comment by marionebl** [comment]
The used asciicast is very wide:

```json
{
  "width": 318,
  "height": 77
}
```

The window traffic lights scale relative to the window width, this reveals they should clamp to a max-size.

**Comment by adius** [comment]
Yeah I think it should definitely fixed (just like in real windows =P).
Allowing the users to change the size sounds cool too (Although I don't really know why someone would want to use that ^^)

**Comment by marionebl** [comment]
Fixed via version 1.1.5 of `svg-term`
